### DISCUSSION TITLE
LFX Mentorship '22 term 03-Sept-Nov

### DISCUSSION THREAD

**Discussion by nate-double-u** [cncf/mentoring#677] [Announcements]
Hello everyone!

The new LFX Mentorship term (03-Sept-Nov) is now open for project ideas: https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2022/03-Sept-Nov

Since I'm a bit late with this announcement, we'll extend the project proposals deadline till August 12, 5:00 PM PDT -- sorry about that, and thanks to everyone who've already made proposals!

Project submission and application timeline:
	• project proposals: August 1 - August 12, 5:00 PM PDT
	• mentee applications open: August 15 - August 24, 5:00 PM PDT
	• application review/admission decisions/HR paperwork: August 25 - September 1, 5:00 PM PDT

We're looking forward to seeing all the project ideas you're interested in working on over the summer!

# Timeline

Mentorship duration - three months (12 weeks - full-time schedule)

| activity | date |
| --- | --- |   
| project proposals | August 1 - August 12, 5:00 PM PDT |
| mentee applications open | August 15 - August 24, 5:00 PM PDT |
| application review/admission decisions/HR paperwork | August 25 - September 1, 5:00 PM PDT |
| Mentorship program begins with the initial work assignments | September 5 (Week 1) | 
| Midterm mentee evaluations and first stipend payments | October 6 (End of Week 5) |
| Final mentee evaluations and mentee feedback/blog submission due, second and final stipend payment approvals | November 16, 5:00 PM PST (End of Week 11) |
| Last day of term | November 30 |

[original announcement email](https://lists.cncf.io/g/tag-cs-mentoring-wg/message/1)

**Comment by abs007** [comment]
Hi, are there any guidelines to keep in mind while applying for more than one project ideas?

**Reply by nate-double-u** [reply]
Hi @abs007, Communication is key. When the applications are opened, each project will have a link to an upstream issue. Take a look, ask questions. The mentors are all happy to discuss the projects. You can apply to multiple projects, but you can only be accepted for one.

Program guidelines: https://github.com/cncf/mentoring/blob/main/lfx-mentorship/README.md#program-guidelines

**Reply by abs007** [reply]
@nate-double-u  So on the rare occasion that I do get selected for 2 projects, will I be rejected from one project from the maintainer side or will I have to do it myself?

**Reply by nate-double-u** [reply]
In the case where two projects want your participation, they'll discuss among themselves and make a decision. For the CNCF LFX Mentorship program this is done in the background. You'll only be offered one position, and won't need to decline any other project.

Note that there is an application limit with the LFX Mentorship program, please only [apply to a maximum of three (3) mentorship programs in one term](https://docs.linuxfoundation.org/lfx/mentorship/mentees).

**Reply by abs007** [reply]
Thank you for your response

**Reply by abs007** [reply]
@nate-double-u another query. Is there any location specific issue during the duration of the program? Like say I switch my location once between uk and india during the course of the program so will there be any issue or do I need to ask this to lfx?

**Comment by nate-double-u** [comment]
Hello Everyone,

Applications are now open for CNCF LFX Mentorship '22 Term 3: Sept-Nov!

https://github.com/cncf/mentoring/tree/main/lfx-mentorship/2022/03-Sept-Nov

Mentee applications open through to August 24, 5:00 PM PDT. Good luck everyone!

**Comment by anutosh491** [comment]
Hello @nate-double-u , Anutosh here . So I have already applied for a project on the LFX Mentorship portal and was thinking if I could apply for another one . The deadline for applying seemingly was 24rth but I see that the portal is still accepting applications . If I submit an application now , will it be counted ? I am not sure about this hence thought of asking !

**Reply by nate-double-u** [reply]
Hi @anutosh491 , sorry for the delayed response. 

It's always best to get an application in before the deadline. Each program and mentor makes their own choices, not the platform, so submitting an application late means they may have already done their reviews, missing anything that has come in after (even if the platform is still accepting applications for whatever reason).
